packages/yum/groonga.spec.in

```diff
@@ -56,6 +57,7 @@ Summary:	Runtime libraries for Groonga
 Group:		System Environment/Libraries
 License:	LGPLv2 and (MIT or GPLv2)
 Requires:	arrow@APACHE_ARROW_MAJOR_VERSION@-libs
+Requires:	libedit
```

Comment:
This should exist in `%package bin` not `%package libs` because `groonga` not `libgroonga.so` needs `libedit.so`.

Reply:
Ah, we don't have `%package bin`.
This should exist in the top-level `Requires`.

Reply:
https://github.com/groonga/groonga/pull/1854/commits/1273f2d747aeaae2c8d66bb17dd88a3532ff19c0